plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/adapter/controllers/CDK3LaunchController.java

```diff
@@ -98,12 +98,23 @@ static String getProfileString(IServer server) {
 		}
 		return profiles;
 	}
+	
+	protected IServer getServerFromLaunch(ILaunchConfigurationWorkingCopy wc) throws CoreException {
```

Comment:
I dont see the benefit of this additional method.

Reply:
I think I had originally put it there for the test to override, but it became unnecessary. Still, doesn't hurt anything.